developer-docs-site/docs/concepts/base-gas.md

```diff
@@ -257,50 +256,22 @@ As of the time of this writing, `min_price_per_gas_unit` in [`transaction.rs`](h
-| Per item   | Create    | $c_i$  | 300,000              |
-| Per item   | Write     | $w_i$  | 300,000              |
-| Per byte   | Read      | $r_b$  | 300                  |
-| Per byte   | Create    | $c_b$  | 5,000                |
-| Per byte   | Write     | $w_b$  | 5,000                |
-
-Maximum amounts are 100 times the minimum amounts, which means that for a utilization ratio of 40% or less, total gas costs will be on the order of 1 to 1.5 times the minimum amounts (see [`base_8192_exponential_curve()`] for supporting calculations).
-Hence, in terms of octas, initial mainnet gas costs can be estimated as follows (divide internal gas by scaling factor, then multiply by minimum gas price):
-
-| Operation       | Operation | Minimum octas |
-|-----------------|-----------|---------------|
-| Per-item read   | $r_i$     | 3000          |
-| Per-item create | $c_i$     | 50,000          |
-| Per-item write  | $w_i$     | 3000          |
-| Per-byte read   | $r_b$     | 3             |
-| Per-byte create | $c_b$     | 50            |
-| Per-byte write  | $w_b$     | 50            |
-
-Here, the most expensive per-item operation by far is creating a new item (via either `move_to<T>()` or adding to a table), which costs nearly 17 times as much as reading or overwriting an old item: $c_i = 16.\overline{6} r_i = 16.\overline{6} w_i$. Additionally:
+### Storage Fee
 
-* Writes cost the same as reads on a per-item basis: $w_i = r_i$
-* On a per-byte basis, however, writes cost the same as creates: $w_b = c_b$
-* Per-byte writes and creates cost nearly 17 times as much as per-byte reads: $w_b = c_b = 16.\overline{6} r_b$
-* Per-item reads cost 1000 times as much as per-byte reads: $r_i = 1000 r_b$
-* Per-item creates cost 1000 times as much as per-byte creates: $c_i = 1000 c_b$
-* Per-item writes cost 60 times as much as per-byte writes: $w_i = 60 w_b$
+When the network load is low, the gas unit price is expected to be low, as a result all of other aspects of the transaction cost are expected to be cheap. Except for the storage fee, which is priced in terms of absolute APT value. Usually the transaction fee is the dominant aspect of the whole transaction cost, as long as the transaction allocates state slots or write to large state items, emit many or large events, or the transaction itself is large -- all of which occupy disk space on all Aptos nodes, hence charged accordingly.
 
-Hence per-item operations cost 1000 times more than per-byte operations for both reads and creates, but only 60 times more for writes.
+On the other hand, releasing state slots by deleting state items are incentivized by the storage refund. For now state slot fee is fully refunded uppon slot deallocation, and the excess state byte fee is not refundable. This is to be changed shortly by distinguishing permanent bytes (those in the global state) and reletive ephemeral bytes (those goes through the ledger history).
```

Comment:
```suggestion
On the other hand, the storage refund incentivizes releasing state slots by deleting state items. The state slot fee is fully refunded upon slot deallocation, while the excess state byte fee is non-refundable. This will soon change by differentiating between permanent bytes (those in the global state) and relative ephemeral bytes (those that traverse the ledger history).
```